js/hole-tabs.tsx

```diff
@@ -8,7 +8,7 @@ import diffTable        from './_diff';
 import { $, $$, comma } from './_util';
```

Comment:
This PR needs a rebase before I can test it, the diff fails to apply here because 8e60ff0 added debounce, not sure why GitHub thinks it could be applied, maybe I need better flags to `git apply` :man_shrugging: